expression/builtin_like_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/pingcap/tidb/util/chunk"
 	"github.com/pingcap/tidb/util/collate"
 	"github.com/pingcap/tidb/util/testutil"
+	"sync"
```

Comment:
Please group the standard library and the third-party library.

Reply:
ok